src/ert_shared/models/base_run_model.py

```diff
@@ -302,13 +302,14 @@ def run_ensemble_evaluator(
             self.ert().resConfig(),
         ).build()
 
+        ensemble.id = str(uuid.uuid1()).split("-", maxsplit=1)[0]
```

Comment:
Would be better if this can be set as part of the builder I believe.

Reply:
I agree. It shouldn't leak into `base_run_model`.

Reply:
I think the result of the discussion https://github.com/equinor/ert/pull/3724#discussion_r947617303 will affect how this should look. Let's resolve that first.